docs/getting-started/quickstart.js-backend.mdx

```diff
@@ -423,7 +453,9 @@ The [`Auth`](/docs/reference/backend/types/auth-object) object contains importan
         import type { Route } from './+types/profile'
 
         export async function loader(args: Route.LoaderArgs) {
-          // Use `getAuth()` to access `isAuthenticated` and the user's ID
+          // The `Auth` object gives you access to properties like `isAuthenticated` and `userId`
+          // Accessing the `Auth` object differs depending on the SDK you're using
+          // https://clerk.com/docs/reference/backend/types/auth-object#how-to-access-the-auth-object
```

Comment:
I like the helpfulness of adding a link here, but they are going to go unchecked if/when we do updates or redirects. 

Because of that, should we leave them or remove them?